actix-http/src/service.rs

```diff
@@ -59,6 +59,7 @@ where
     S::Response: Into<Response<B>> + 'static,
     <S::Service as Service<Request>>::Future: 'static,
     B: MessageBody + 'static,
+    B::Error: Into<Error>,
```

Comment:
having bounds look like this (for now) is a pretty well compatible change since trivially: `Error: Into<Error>`